lib/src/install.rs

```diff
@@ -1570,7 +1589,15 @@ pub(crate) async fn install_to_disk(mut opts: InstallToDiskOpts) -> Result<()> {
         (rootfs, loopback_dev)
     };
 
-    install_to_filesystem_impl(&state, &mut rootfs).await?;
+    install_to_filesystem_impl(&state, &mut rootfs, Cleanup::Skip).await?;
+
+    // Finalize mounted filesystems
+    if !rootfs.skip_finalize {
```

Comment:
Don't we also need to delete the copy of this in `install_to_filesystem_impl`?

Reply:
ah whoops, forgot to push removing those extra calls I was trying out. Updated now.